interfaces/builtin/unity7.go

```diff
@@ -80,22 +78,29 @@ owner @{HOME}/.local/share/fonts/{,**} r,
 /usr/share/icons/*/index.theme             rk,
 /usr/share/pixmaps/                        r,
 /usr/share/pixmaps/**                      r,
-/usr/share/unity/icons/**                  r,
```

Comment:
This access is removed but not added anywhere back AFAICT - is there a regression risk here?

Reply:
No. This path does not exist in core images (notice is is in /usr, which comes from core* snaps). This is explained in commit 105b47f2f9b8e07324fcc1714a8071a4d7031ea8:

```
    interfaces/unity7: limit access to /usr/share
    
    Currently, the unity7 interface allows access to all of
    /usr/share/applications. The files in /usr/share/applications come from
    the core snaps, and with the exception of files related to xdg-open,
    snaps are unable to do anything useful with these files (since they are
    unable to execute the application). As such, limit the access to:
    
      /usr/share/applications/ r,
      /usr/share/applications/mimeapps.list r,
      /usr/share/applications/xdg-open.desktop r,
    
    and suppress noisy denials by adding explicit deny rules for the desktop
    files known to be shipped in the core, core18 and core20 snaps.
    
    In addition, to avoid confusion, remove legacy rules inherited from
    snappy 15.04 that do not exist in the core, core18 and core20 snaps:
    
      /usr/share/gnome/applications/
      /usr/share/applications/mimeinfo.cache
      /usr/share/unity/icons/**
      /usr/share/thumbnailer/icons/**
      /usr/share/themes/**
      /usr/share/gnome/glib*/schemas/{,*}
      /usr/share/ubuntu/glib*/schemas/{,*}
    
    References:
    https://bugs.launchpad.net/snapd/+bug/1868051 (related)
```